lib/api/accounts.js

```diff
@@ -190,26 +216,11 @@ module.exports = function (app) {
 		};
 
 		this.getVoters = (account, cb) => {
-			if (!account.publicKey) {
+			if (!account.address) {
 				return cb(null, account);
 			}
-			return request.get({
-				url: `${app.get('lisk address')}/voters?publicKey=${account.publicKey}`,
-				json: true,
-			}, (err, response, body) => {
-				if (err || response.statusCode !== 200) {
-					return cb(err || 'Response was unsuccessful');
-				} else if (body && Array.isArray(body.data)) {
-					account.voters = Array.isArray(body.data.voters)
-						&& body.data.voters.length > 0 ? body.data.voters : null;
-					_.each(account.voters, (d) => {
-						d.knowledge = knowledge.inAccount(d);
-					});
-				} else {
-					account.voters = null;
-				}
-				return cb(null, account);
-			});
+			account.voters = [];
+			return this.getVotersCall(account, 100, 0, cb);
```

Comment:
We should avoid using literals as parameters. The literals above should be replaced with defined variables or constants. ex. `const limit = 100` and reference to the `limit` in the function params.